### ISSUE TITLE
Dpkg ownership error

### ISSUE THREAD

**Issue by BoxDroppingManApe** [Rydra/bgm-for-es#3]
I'm getting an error on the latest RetroPie Stretch image when trying to install this:

`dpkg: error processing archive /var/cache/apt/archives/python-es-bgm_1.0.1-1_all.1-1-all (--unpack):
 error setting ownership of '/home/pi/RetroPie/roms/music.dpkg-new': Operation not permitted`

**Comment by alan2022000** [comment]
I was getting some problems too. 

I’m still a newbie to all this. (Mainly Linux stuff. Lol). But I managed to get it to install. 

Here’s the error I was getting. 

error setting ownership of './home/pi/RetroPie/roms/music/PLACE_YOUR_MUSIC_HERE': Operation not permitted
Disabling background music daemon...
Failed to disable unit: No such file or directory
dpkg: error while cleaning up:
 subprocess new post-removal script returned error exit status 1
Errors were encountered while processing:
 /var/cache/apt/archives/python-es-bgm_1.0.1-1_all.1-1-all
E: Sub-process /usr/bin/dpkg returned an error code (1)


What worked for me was the fact that my music folder is on a usb drive. (FAT32). To what I have read it seems to be that FAT can not have permissions so that’s why it errors. 

All I did was shutdown my pi. Unplugged the usb stick. Turned on and did the install command again. And it installed without problem.  

So I did sudo shutdown, plugged usb stick back in. Turned it On. 

I already had some mp3’s In the music folder and it just started playing once emulationstation started. 

Hope this helps anyone else that has the retropie folder mounted via usb.
